### PR TITLE
fix some translations issues mentioned on Transifex

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
* There is no local handbook (anymore), only online, and F1 opens that.
* When borrowing words from a foreign language, local grammar applies, not foreign, so it is "Glissandos" (English plural s), not "Glissandi" (Latin/Italian plural). Alternative might be to change "Arpeggios" to Arpeggii".
* Better tooltip for loop playback (copy from shortcut dialog).
* Remove apparent copy/paste artifact from Aeolus dialog, thus removing a couple strings from translation. May need more work, either removal of Aeolus and ist tab all together, or marking most of íts strings to not get translated.
* Fix two short instrument names. Not sure whether also to change the "Piccolo Trumpet in X" to "X Piccolo Trumpet". At least for the one in A this would look rather strange... (Article vs. Notename)

### CODE REVIEW

File: mscore/shortcut.cpp [hunk 1]
```diff
@@ -31,10 +31,10 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::MAIN_WINDOW,
          STATE_ALL,
-         "local-help",
-         QT_TRANSLATE_NOOP("action","Local Handbook..."),  // Appears in menu
-         QT_TRANSLATE_NOOP("action","Local handbook"),     // Appears in Edit > Preferences > Shortcuts
-         QT_TRANSLATE_NOOP("action","Show local handbook"), // Appears if you use Help > What's This?
+         "help",
```
Comment (lasconic) [inline mscore/shortcut.cpp]: If you change this, you need to change shortcuts.xml as well.

Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: Ah, yes., gimme 5 minutes ;-)

Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: done

**Comment by Jojo-Schmitz** [comment]
I believe this applies to master and 2.0.4, esp. as I took the issue from the current translations which is for 2.0.x

**Comment by lasconic** [comment]
mmm I'm not sure about the help shortcut... if someone customized the shortcut it will probably do a mess.

**Comment by Jojo-Schmitz** [comment]
true, so better leave the name for that shortcut local-help, just change the user visible text

**Comment by lasconic** [comment]
Could you do a new PR then, on 2.0.4?

**Comment by Jojo-Schmitz** [comment]
Will try, later. Might be easier to cherry pick this one and fix the very few lines in a 2nd commit?

**Comment by Jojo-Schmitz** [comment]
There we go, PR #2779

**Comment by lasconic** [comment]
thanks

**Pull request merged**